feature-detects/audio/webaudio.js

```diff
@@ -8,7 +8,7 @@
   "builderAliases": ["audio_webaudio_api"],
   "authors": ["Addy Osmani"],
   "notes": [{
-    "name": "W3 Specification",
+    "name": "W3 Spec",
```

Comment:
any particular reason for this?

Reply:
was the only occasion where "Specification" was used, all other occurences were "W3 Spec" Just my little tidiness-spleen